pyaedt/edb_grpc/core/edb_data/components_data.py

```diff
@@ -0,0 +1,962 @@
+                return "0"
+            for pinpair in pinpairs:
+                pair = model.GetPinPairRlc(pinpair)
+                return pair.L.ToString()
+        return None
+
+    @ind_value.setter
+    def ind_value(self, value):  # pragma no cover
+        if value:
+            if self.rlc_values == [None, None, None]:
+                self.rlc_values = [0, value, 0]
+            else:
+                self.rlc_values = [self.rlc_values[1], value, self.rlc_values[2]]
+
+    @property
+    def is_parallel_rlc(self):
+        """Define if model is Parallel or Series.
+
+        Returns
+        -------
+        bool
+            ``True`` if it is a parallel rlc model. ``False`` for series RLC. ``None`` if not an RLC Type.
+        """
+        cmp_type = int(self.edbcomponent.GetComponentType())
+        if 0 < cmp_type < 4:
+            model = self.component_property.GetModel().Clone()
+            pinpairs = model.PinPairs
+            for pinpair in pinpairs:
+                pair = model.GetPinPairRlc(pinpair)
+                return pair.IsParallel
+        return None
+
+    @is_parallel_rlc.setter
+    def is_parallel_rlc(self, value):  # pragma no cover
+        if not len(self._pin_pairs):
+            logging.warning(self.refdes, " has no pin pair.")
+        else:
+            if isinstance(value, bool):
+                componentProperty = self.edbcomponent.GetComponentProperty()
+                model = componentProperty.GetModel().Clone()
+                pinpairs = model.PinPairs
+                if not list(pinpairs):
+                    return "0"
+                for pin_pair in pinpairs:
+                    pin_pair_rlc = model.GetPinPairRlc(pin_pair)
+                    pin_pair_rlc.IsParallel = value
+                    pin_pair_model = self._edb_model
+                    pin_pair_model.SetPinPairRlc(pin_pair, pin_pair_rlc)
+                    comp_prop = self.component_property
+                    comp_prop.SetModel(pin_pair_model)
+                    self.edbcomponent.SetComponentProperty(comp_prop)
+
+    @property
+    def center(self):
+        """Compute the component center.
+
+        Returns
+        -------
+        list
+        """
+        center = self.component_instance.GetCenter()
+        return [center.X.ToDouble(), center.Y.ToDouble()]
+
+    @property
+    def bounding_box(self):
+        """Component's bounding box.
+
+        Returns
+        -------
+        List[float]
+            List of coordinates for the component's bounding box, with the list of
+            coordinates in this order: [X lower left corner, Y lower left corner,
+            X upper right corner, Y upper right corner].
+        """
+        bbox = self.component_instance.GetBBox()
+        pt1 = bbox.Item1
+        pt2 = bbox.Item2
+        return [pt1.X.ToDouble(), pt1.Y.ToDouble(), pt2.X.ToDouble(), pt2.Y.ToDouble()]
+
+    @property
+    def rotation(self):
+        """Compute the component rotation in radian.
```

Comment:
Same component about not using a verb to start a property description. It should probably be "Component rotation in radian."
Is it radian or radians?